src/sdk/accountspace_nb.js

```diff
@@ -63,7 +63,11 @@ class AccountSpaceNB {
     }
 
     async get_user(params, account_sdk) {
-
+        // Added to fix the coretest, ListAttachedUserPoliciesCommand 
+        // calling get_user with more param we allow and its failing RPC schema validation.
+        params = {
+            username: params.username,
+        };
```

Comment:
It is not clear, in the file `src/endpoint/iam/ops/iam_get_user.js` we pass only the `username` why do you see more params?
https://github.com/noobaa/noobaa-core/blob/d3c091e0b95197f772c7703c478c56947f2bf325/src/endpoint/iam/ops/iam_get_user.js#L14-L20